lib/DDG/Spice/Github.pm

```diff
@@ -16,10 +16,16 @@ triggers startend => "github";
 spice to => 'https://api.github.com/legacy/repos/search/$1?callback={{callback}}';
 spice proxy_cache_valid => '200 30d';
 
-handle query_lc => sub {
-    s/^github\s+|\s+github$//;
+handle remainder_lc => sub {
+
+	s/^github\s+|\s+github$//;
```

Comment:
This line can be removed as `github` will not be included in `$_` as we're only using the `remainder`